docs/frequently-asked-questions.rst

```diff
@@ -317,7 +317,8 @@ by `msg.data`.
 Can state variables be initialized in-line?
 ===========================================
 
-Yes, this is possible for most types (even for structs), but not for arrays.
+Yes, this is possible for all types (even for structs). However, for arrays it 
+should be noted that you must declare them as static memory arrays. Futhermore, multi dimensional arrays cannot be declared inline.
```

Comment:
It should actually work for multi-dimensional arrays. Did you try
`uint8[4][4] memory dyn = [[1, 2, 3, 4], [2, 3, 4, 5], [3, 4, 5, 6], [4, 5, 6, 7]];` in the test at the end (i.e. `uint8` instead of `uint`)?